docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+
+<!-- markdownlint-disable MD046 -->
+
+=== "deployment.yaml"
+
+    ```yaml
+    {% include "./assets/keda/sample-app.yaml" %}
+    ```
+
+=== "service.yaml"
+
+    ```yaml
+    {% include "./assets/keda/sample-service.yaml" %}
+    ```
+
+<!-- markdownlint-enable MD046 -->
+
+Please create a `podtato-kubectl` namespace and apply the above manifests
+to your cluster and continue with the next steps.
+After applying, use the following command to ensure that the application is up and running:
+
+```shell
+$ kubectl get pods -n podtato-kubectl
+podtato-head-entry-58d6485d9b-ld9x2         1/1     Running     (2m ago)
+```
+
+## Create KeptnMetric and KeptnMetricsProvider custom resources
+
+To be able to react on the metrics of our application, we need to create
+`KeptnMetrics` and `KeptnMetricsProvider` custom resources.
+These metrics are
+exposed via the Keptn Metrics Operator, which gives us the possibility to configure
+KEDA to react on the values of these metrics:
+
+<!-- markdownlint-disable MD046 -->
+
+=== "KeptnMetric"
+
+    ```yaml
+    {% include "./assets/keda/keptnmetric.yaml" %}
+    ```
+
+=== "KeptnMetricsProvider"
+
+    ```yaml
+    {% include "./assets/keda/keptnmetricsprovider.yaml" %}
+    ```
+
+<!-- markdownlint-enable MD046 -->
+
+For more information about the `KeptnMetric` and `KeptnMetricsProvider` custom resources,
+please refer to the [CRD documentation](../reference/api-reference/metrics/v1/index.md).
+
+After a few seconds we should be able to see values for the `cpu-throttling` metric:
+
+```shell
+$ kubectl describe  keptnmetrics.metrics.keptn.sh cpu-throttling -n podtato-kubectl
+Name:         cpu-throttling
+Namespace:    podtato-kubectl
+API Version:  metrics.keptn.sh/v1
+Kind:         KeptnMetric
+Spec:
+  Fetch Interval Seconds:  60
+  Provider:
+    Name:  prometheus-provider
+  Query:  avg(rate(container_cpu_cfs_throttled_seconds_total{container="server", namespace="podtato-kubectl"}))
+Status:
+  Raw Value: <omitted for readability>
+  Value:         4.53
+```
+
+Here we can see that the value of the `cpu-throttling` metric is `4.53`
+
+## Set up the KEDA ScaledObject
+
+Now that we are able to retrieve the value of our metric, and have it stored in
+our cluster in the status of our `KeptnMetric` custom resource, we can configure
+a `ScaledObject` to make use of this information in KEDA and therefore scale
+our application automatically:
+
+```yaml
+{% include "./assets/keda/scaledobject.yaml" %}
+```
+
+As we can see in this example, by setting the `url` field,
+we are now referring to the `KeptnMetric` and fetch it from the
+Keptn Metrics Operator.
+KEDA will scale up our application, until our target value or `1` is reached,
+or we hit the maximum number of replicas which is `3` in this example.
+
+If the load of the application is high enough, we will be able to see
+the automatic scaling of our application:
+
+```shell
+$ kubectl describe scaledobject -n podtato-kubectl my-scaledobject
```

Comment:
let's try to put this to a separate file

Reply:
done